docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+services:
+  new_fastapi_db:
+    image: postgres:16
+    container_name: new_fastapi_db_container
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER:-default_user}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD:-default_password}
+      POSTGRES_DB: ${POSTGRES_DB:-default_db}
+    ports:
+      - "5433:5432"
+    volumes:
+      - new_postgres_data:/var/lib/postgresql/data
```

Comment:
When you run the service, the tables and all the data will be stored in the new_postgres_data folder. If you want to recreate everything from scratch, you can delete that folder

Reply:
Thanks for this, I was searching about it